### DISCUSSION TITLE
How to avoid task error on windows

### DISCUSSION THREAD

**Discussion by markdemich** [BowlerHatLLC/vscode-as3mxml#672] [Q&A] [answered]
I created a task in my repo for use on the mac because I needed to add a dependsOn. 
```
Error: The actionscript task detection didn't contribute a task for the following configuration:
{
    "type": "actionscript",
    "debug": false,
    "air": "mac",
    "group": "build",
    "problemMatcher": [],
    "label": "Build XXXX.app",
    "presentation": {
        "echo": true,
        "reveal": "always",
        "focus": true,
        "panel": "shared",
        "showReuseMessage": true,
        "clear": false
    },
    "dependsOn": "Create Deploy Folder"
}
The task will be ignored.
```
However, I get an error popup whenever I click CTRL-SHIFT-B on windows.  Is there a way to avoid this?

**Comment by joshtynjala** [comment]
Assuming that you also need a build task for Windows, maybe you could do something like this instead:

```jsonc
{
    "type": "actionscript",
    "debug": false,
    "windows": {
        "air": "windows",
        "label": "Build XXXX.exe"
    },
    "osx": {
        "air": "mac",
        "label": "Build XXXX.app"
    },
   // ...
}
```

**Reply by markdemich** [reply]
Unfortunately, that doesn't work.  It ignores that part.  I think you would need to change the "actionscript" task to support that.

**Reply by joshtynjala** [reply, marked as answer]
I think I'm going to add a new `air` value: `"bundle"`. It will be equivalent to either `"windows"` or `"mac"`, depending on the operating system VSCode is running on.
